ArduCopter/mode.cpp

```diff
@@ -405,33 +405,15 @@ void Mode::get_pilot_desired_lean_angles(float &roll_out_cd, float &pitch_out_cd
         pitch_out_cd = 0.0;
         return;
     }
-    // fetch roll and pitch stick positions
-    float thrust_angle_x_cd = - channel_pitch->get_control_in();
-    float thrust_angle_y_cd = channel_roll->get_control_in();
 
-    // limit max lean angle
-    angle_limit_cd = constrain_float(angle_limit_cd, 1000.0f, angle_max_cd);
+    //transform pilot's normalised roll or pitch stick input into a roll and pitch euler angle command
+    float roll_out_deg;
+    float pitch_out_deg;
+    rc_input_to_roll_pitch(roll_out_deg, pitch_out_deg, channel_roll->norm_input_dz(), channel_pitch->norm_input_dz(), angle_max_cd * 0.01,  angle_limit_cd * 0.01);
```

Comment:
This change from `get_control_in` to `norm_input_dz` brakes simple and supper simple. 

You have to manually normalize the control in method for the time being.

Reply:
Thanks Pete! Nice catch.